src/ui/components/phoneNumber/PhoneNumber.scss

```diff
@@ -0,0 +1,13 @@
+@import "../../styles/mixin.scss";
+
+.phone-display {
+  display: flex;
+  align-items: center;
+  gap: 5px;
+  @include gap-ios(5px);
```

Comment:
I would prefer if we just dont use gap instead. Doing it this way just adds more boilerplate and theres a high chance that someone unfamiliar with this project will just miss out on adding the mixin